backend/app/tests/services/collections/test_create_collection.py

```
@@ -33,30 +34,33 @@ def aws_credentials() -> Any:
-            id=job_id,
-            action_type=CollectionActionType.CREATE,
-            project_id=project.id,
-            collection_id=None,
-            status=CollectionJobStatus.PENDING,
-        )
-    )
+def _mock_provider_with_size(llm_service_id: str, llm_service_name: str):
+    """Returns a mock provider whose upload_files sets file_size_kb=10.0 on each doc."""
+    mock_provider = get_mock_provider(llm_service_id, llm_service_name)
+
+    def _set_file_size(storage, docs, project_id):
```

Ruff (0.15.12)

[warning] Unused function argument: `storage`

(ARG001)